tests/functional/spec/household-composition.spec.js

```diff
@@ -0,0 +1,114 @@
+
+import chai from 'chai'
+import {startQuestionnaire} from '../helpers'
+import HouseholdCompositionPage from '../pages/surveys/household_composition/household-composition.page'
+import HouseholdCompositionSummary from '../pages/surveys/household_composition/summary.page'
+
+const expect = chai.expect
+
+describe('Household composition question for census test.', function() {
+
+  var household_composition_schema = 'test_household_question.json';
+
+  it('Given no people added, when enter a name and submit, then name should be displayed on summary.', function() {
+    //Given
+    startQuestionnaire(household_composition_schema)
+
+    //When
+    HouseholdCompositionPage.setPersonName(0, 'Person One').submit()
+
+    // Then
+    expect(HouseholdCompositionSummary.isNameDisplayed('Person One')).to.be.true
+  })
+
+  it('Given no people added, when enter a name and add another person, then there should be two input fields displayed.', function() {
+    //Given
+    startQuestionnaire(household_composition_schema)
+
+    //When
+    HouseholdCompositionPage.setPersonName(0, 'Person One').addPerson()
+
+    // Then
+    expect(HouseholdCompositionPage.isInputVisible(0)).to.be.true
+    expect(HouseholdCompositionPage.isInputVisible(1)).to.be.true
+  })
+
+  it('Given two people added, when submitted, both names should appear on summary.', function() {
+    //Given
+    startQuestionnaire(household_composition_schema)
+
+    //When
+    HouseholdCompositionPage
+        .setPersonName(0, 'Person One')
+        .addPerson()
+        .setPersonName(1, 'Person Two')
+        .submit()
+
+    // Then
+    expect(HouseholdCompositionSummary.areNamesDisplayed(['Person One', 'Person Two'])).to.be.true
+  })
+
+  it('Given three people added, when submitted, all three names should appear on summary.', function() {
```

Comment:
Maybe we don't need this test as it doesn't exercise anything more than `Given two people added, when submitted, both names should appear on summary.`